src/Stratis.Bitcoin.Features.RPC/RPCClient.cs

```diff
@@ -415,37 +428,6 @@ public static string TryGetDefaultCookieFilePath(Network network)
             return defaultPaths.TryGetValue(network, out string path) ? path : null;
         }
 
-        private static Uri BuildUri(string hostOrUri, int port)
```

Comment:
Why was this method removed? I don't see the logic replicated.

Reply:
@dangershony I believe this is now unused because of his changes.

Reply:
Also, .NET has a `UriBuilder` class ;)

Reply:
In the `RPCClientFactory.cs` we were getting a string from URI which in turn was converted back to URI using the BuildUri method.